data/35e/wizards_of_the_coast/core/players_handbook/ph_spells.lst

```diff
@@ -1,7 +1,7 @@
 # CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformated by prettylst.pl v1.51 (build 24906)
 
 # Spell Name
-YYY
+
```

Comment:
Why? What did the triple Y ever do to deserve to be deleted? As a side note, that unassuming, unassigned, lacking any details spell has been instrumental in making some classes work.

Reply:
... thought that was a typo I'd introduced! Sorry,  Added it back in, and added comment indicating it is necessary.